### PR TITLE
CHANGELOG for 0.16.0

### PR DESCRIPTION
**Pull request by datho7561**
Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

**Comment by fbricon** [comment]
license changed to EPL 2.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+### Bug Fixes
+
+ * Wait for the language server to stop before exiting. See [#437](https://github.com/redhat-developer/vscode-xml/pull/437).
+ * Use `User-Agent: LemMinX` when downloading schemas to prevent HTTP 403 when using Java 8. See [#429](https://github.com/redhat-developer/vscode-xml/issues/429).
+ * Fix rename not completing when using the binary. See [#424](https://github.com/redhat-developer/vscode-xml/issues/424).
+ * Fix a regression where the vscode-xml API was not returned when calling extension.ts -> activate(). See [#418](https://github.com/redhat-developer/vscode-xml/issues/418).
+ * Fix URL for extensions. See [#413](https://github.com/redhat-developer/vscode-xml/pull/413).
+ * Notify the user when the language server fails to start. See [#409](https://github.com/redhat-developer/vscode-xml/issues/409).
+ * Fix NPE when hovering on a malformed document. See [eclipse/lemminx#984](https://github.com/eclipse/lemminx/issues/984).
+ * `trimTrailingWhitespace` option is not respected by `textDocument/formatting`. See [#827](https://github.com/eclipse/lemminx/issues/827).
```
Comment (rgrunber) [inline CHANGELOG.md:22]: Should probably be eclipse/lemminx#827

Comment (rgrunber) [reply CHANGELOG.md:22]: Once addressed, I think it's ready to merge.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## [0.16.0](https://github.com/redhat-developer/vscode-xml/milestone/19?closed=1) (April 13, 2021)
+
+### Enhancements
+
+ * Add a progress bar to the binary server download. See [#433](https://github.com/redhat-developer/vscode-xml/issues/433).
+ * Collect Telemetry data. See [#415](https://github.com/redhat-developer/vscode-xml/issues/415).
+ * Add the article about the 0.15.0 release. See [#439](https://github.com/redhat-developer/vscode-xml/pull/439).
```
Comment (fbricon) [inline CHANGELOG.md:9]: doesn't belong in changelog

**Review by rgrunber — APPROVED** [review]

**Pull request merged**